### ISSUE TITLE
[BUG] "linkchecker" fails CI at the step "Fail if there were link errors"

### ISSUE THREAD

**Issue by cliu123** [opensearch-project/OpenSearch#960]
**Describe the bug**
CI fails at the step "Fail if there were link errors", but the errors reported look like false positive.
For example, the CI failed on the [PR#889](https://github.com/opensearch-project/OpenSearch/pull/889/checks), and the following errors are reported.
```
2021-07-13T05:28:55.5195733Z 🚫 Errors...........2
2021-07-13T05:28:55.5196000Z 
2021-07-13T05:28:55.5196347Z Errors in README.md
2021-07-13T05:28:55.5197628Z ✗ https://docs-beta.opensearch.org/ (HTTP status client error (404 Not Found) for url (https://docs-beta.opensearch.org/))
2021-07-13T05:28:55.5198553Z 
2021-07-13T05:28:55.5199466Z Errors in plugins/examples/rest-handler/src/yamlRestTest/resources/rest-api-spec/api/cat.example.json
2021-07-13T05:28:55.5208880Z + echo
2021-07-13T05:28:55.5210365Z + echo ::set-output name=exit_code::2
2021-07-13T05:28:55.5213712Z ✗ https://docs-beta.opensearch.org/opensearch/install/plugins/ (HTTP status client error (404 Not Found) for url (https://docs-beta.opensearch.org/opensearch/install/plugins/))
```
But both links that were errored out work as expected.
https://docs-beta.opensearch.org/
https://docs-beta.opensearch.org/opensearch/install/plugins/

Another example: 
[PR#947](https://github.com/opensearch-project/OpenSearch/pull/947/checks)

**Note**:
The step was introduced by [this PR](https://github.com/opensearch-project/OpenSearch/pull/877) today. 

**Expected behavior**
CI should not fail.

**Plugins**
No plugins are enabled

**Comment by VachaShah** [comment]
Hi @cliu123, I have a PR to fix the links: #957, once that is merged to main, this would not be an issue.

**Comment by dblock** [comment]
Those links now are redirects since launch. It's actually awesome that link checker caught this.
